_component_design/tables-basic.md

```diff
@@ -153,3 +155,71 @@ When using the check box icon within a table, each row supports the following st
 {% endcolumn %}
 
 </div>
+
+## Variations
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Dual line text table
+
+This variation of the basic table pattern is designed to allow for specialized content where cells need to display two lines of content.
+
+#### When to use
+
+This variation is used in the following situations: 
+
+- When a key has two values that are meaningfully related to each other and require display within the same cell. For example, last updated cell in the image to the left.
+- When two independent key:values are more meaningful displayed together.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Dual line text table example. **note to scale**"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-dualline-overview.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+#### Best practices
+
+- **Rule**: (**Important**) Each cell with more than one line of text must have a designated primary and secondary value.
+- **Rule**: In rows with the [table controls]({{site.baseurl}}/components/tables-controls.html) sorting pattern applied, the pattern will always sort based on the primary key.
+- **Suggestion**: Increase the `font-weight` (or make another meaningful distinction ) between the primary and secondary values in dual-line rows to make the sorting function more clear to a user.
+- **Suggestion**: Due to the amount of information presenting with this pattern, it performs best when paired with another pattern that allows for segmenting the table content. For example: [assisted search]({{site.baseurl}}/components/assisted-search.html).
+
+{% endcolumn %}
+	
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Spacing
+
+The dual-line text variation follows all basic table with the following exceptions:
+
+- The top and bottom padding for cells moves from 20px to 16px.
+- Spacing between lines in dual-line rows are 10px apart. 
+	- Suggested implementation: apply 1.5em line height to dual-line rows
+- Line height for wrapped rows is modified to 12px.
+- In lines displaying two independent key:value pairs designate a primary key:value and a secondary key:value. The primary key:value has the `font-weight` increased to **bold**. 
```

Comment:
These aren't really key/value pairs in the rows.  Restate this along the lines of:

`In lines displaying two independent text rows, emphasize the primary line (on top) by increasing its font weight to bold.`

Reply:
updated.